tests/run-make/executable-no-mangle-strip/main.rs

```diff
@@ -0,0 +1,16 @@
+unsafe extern "C" {
```

Comment:
Nit: This can be a UI test, use something like `//@ build-pass` or `//@ run-pass`